src/main/java/com/redhat/quarkus/mandrel/collector/report/model/RunnerInfo.java

```diff
@@ -26,26 +26,29 @@
 import jakarta.persistence.DiscriminatorColumn;
 import jakarta.persistence.DiscriminatorType;
 import jakarta.persistence.Entity;
+import jakarta.persistence.Index;
 import jakarta.persistence.Inheritance;
 import jakarta.persistence.InheritanceType;
 import jakarta.persistence.Table;
 
 @JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
 @Entity
-@Table(name = "runner_info")
+@Table(name = "runner_info", indexes = {
+        @Index(columnList = "testVersion, graalvmVersion, quarkusVersion, jdkVersion, description, ghPR") })
 @Inheritance(strategy = InheritanceType.SINGLE_TABLE)
 @DiscriminatorColumn(discriminatorType = DiscriminatorType.INTEGER, name = "stats_type")
 public class RunnerInfo extends PanacheEntity {
 
     private String testVersion;
-    private String mandrelVersion;
+    private String graalvmVersion;
     private String quarkusVersion;
     private String jdkVersion;
     private String operatingSystem;
     private String architecture;
     private long memorySizeBytes;
     private long memoryAvailableBytes;
     private String description;
+    private String ghPR;
```

Comment:
Should we call this `url` or something, so it's more generic?

Reply:
+1, maybe `trigger`? It could be anything from a string explaining what triggered this run, a url to the github CI run, or a PR/issue url.

Reply:
I swapped it with "triggeredBy" :+1: